src/mon/Monitor.cc

```diff
@@ -3005,9 +3005,7 @@ void Monitor::get_cluster_status(stringstream &ss, Formatter *f)
 	mgrmon()->get_map().print_summary(nullptr, &ss);
 	ss << "\n";
       }
-      if (mdsmon()->get_fsmap().filesystem_count() > 0) {
-	ss << "    mds: " << spacing << mdsmon()->get_fsmap() << "\n";
-      }
+      ss << "    mds: " << spacing << mdsmon()->get_fsmap() << "\n";
```

Comment:
We could modify this to print if filesystem_count() or standby count > 0...?

Reply:
No strong feelings either way; thanks for fixing this (I've been hitting it while test deploying mds daemons without an fs)

Reply:
I didn't consider the case where there are no MDS, I'll double check how that looks.

Reply:
Done